mne/epochs.py

```diff
@@ -2130,7 +2132,8 @@ def to_data_frame(self, picks=None, index=None,
         assert all(len(mdx) == len(mindex[0]) for mdx in mindex)
         # build DataFrame
         df = _build_data_frame(self, data, picks, long_format, mindex, index,
-                               default_index=['condition', 'epoch', 'time'])
+                               default_index=['condition', 'epoch', 'time'],
+                               verbose=verbose)
```

Comment:
There should be no need to pass this param to `_build_data_frame` based on how the magic of `verbose` works
```suggestion
                               default_index=['condition', 'epoch', 'time'])
```